common/src/main/java/tw/edu/ntut/sce/java18/common/dao/impl/RoomTypeDaoImpl_JDBC.java

```diff
@@ -0,0 +1,99 @@
+          roomTypeBean.setQRefrigerator(resultSet.getInt("Q_Refrigerator"));
+          roomTypeBean.setQAirCondition(resultSet.getInt("Q_AirCondition"));
+          roomTypeBean.setQHeater(resultSet.getInt("Q_Heater"));
+          roomTypeBean.setQMirror(resultSet.getInt("Q_Mirror"));
+          roomTypeBean.setQFlowTable(resultSet.getInt("Q_Flow_Table"));
+          roomTypeBean.setQSideTableS(resultSet.getInt("Q_Side_Table_S"));
+          roomTypeBean.setQSideTableL(resultSet.getInt("Q_Side_Table_L"));
+          roomTypeBean.setQBedBoardS(resultSet.getInt("Q_Bed_Board_S"));
+          roomTypeBean.setQBedBoardL(resultSet.getInt("Q_Bed_Board_S"));
+          roomTypeBean.setQDeskS(resultSet.getInt("Q_Desk_S"));
+          roomTypeBean.setQDeskL(resultSet.getInt("Q_Desk_L"));
+          roomTypeBean.setQBedS(resultSet.getInt("Q_Bed_S"));
+          roomTypeBean.setQBedL(resultSet.getInt("Q_Bed_L"));
+          roomTypeBean.setQWindowScreenS(resultSet.getInt("Q_Window_Screen_S"));
+          roomTypeBean.setQWindowScreenL(resultSet.getInt("Q_Window_Screen_L"));
+          roomTypeBean.setQWindowScreenG(resultSet.getInt("Q_Window_Screen_G"));
+          roomTypeBean.setQWardrobe(resultSet.getInt("Q_Wardrobe"));
+          roomTypeBean.setQChair(resultSet.getInt("Q_Chair"));
+
+          roomTypeBean.setPTV(resultSet.getInt("Q_TV"));
+          roomTypeBean.setPRefrigerator(resultSet.getInt("P_Refrigerator"));
+          roomTypeBean.setPAirCondition(resultSet.getInt("P_AirCondition"));
+          roomTypeBean.setPHeater(resultSet.getInt("P_Heater"));
+          roomTypeBean.setPMirror(resultSet.getInt("P_Mirror"));
+          roomTypeBean.setPFlowTable(resultSet.getInt("P_Flow_Table"));
+          roomTypeBean.setPSideTableS(resultSet.getInt("P_Side_Table_S"));
+          roomTypeBean.setPSideTableL(resultSet.getInt("P_Side_Table_L"));
+          roomTypeBean.setPBedBoardS(resultSet.getInt("P_Bed_Board_S"));
+          roomTypeBean.setPBedBoardL(resultSet.getInt("P_Bed_Board_S"));
+          roomTypeBean.setPDeskS(resultSet.getInt("P_Desk_S"));
+          roomTypeBean.setPDeskL(resultSet.getInt("P_Desk_L"));
+          roomTypeBean.setPBedS(resultSet.getInt("P_Bed_S"));
+          roomTypeBean.setPBedL(resultSet.getInt("P_Bed_L"));
+          roomTypeBean.setPWindowScreenS(resultSet.getInt("P_Window_Screen_S"));
+          roomTypeBean.setPWindowScreenL(resultSet.getInt("P_Window_Screen_L"));
+          roomTypeBean.setPWindowScreenG(resultSet.getInt("P_Window_Screen_G"));
+          roomTypeBean.setPWardrobe(resultSet.getInt("P_Wardrobe"));
+          roomTypeBean.setPChair(resultSet.getInt("P_Chair"));
+        }
+      }
+    } catch (SQLException ex) {
+      ex.printStackTrace();
+      throw new RuntimeException("RoomTypeDaoImpl_JDBC類別#queryRoomType()發生例外: " + ex.getMessage());
+    }
+    return roomTypeBean;
+  }
+
+  @Override
+  public RoomTypeBean updateRoomTypeInfo(String type) {
+    return null;
```

Comment:
1. 方法沒用到，不需要先加在這個 PR
2. 如果一定要先加這個方法，但等人實作，可以改成這樣

```suggestion
    throw new UnsupportedOperationException();
```